doc/installing/README.md

```diff
@@ -1,46 +1,88 @@
-script](universal.md) which works for any platform and for any version
-of the engine.
+The OpenQuake Engine can be installed in several different ways.
+The engine runs on Linux, macOS and Windows; on laptops, workstations, standalone servers and multi-node clusters.
 
-If you are on Windows and you do not need to interact with any other Python software you may also use the **[self-installable binary distribution for Windows 10](windows.md)** (available only for LTS versions). Administrator level access may be required.
-If you are on Linux and you do not need to interact with any other Python software you may also use the **binary packages for [Ubuntu](ubuntu.md)** or **[RedHat (and clones)](rhel.md)** (available only for LTS versions) that require root access.
+> **Warning**:
+> If you already have an engine installation, before installing the new version you must [uninstall](universal.md#uninstalling-the-engine) the old one.
 
-The binary distributions of the engine include their own Python, so they are totally separated by other Python software you may have. The universal installer uses the system Python, but installs the engine in its own virtual environment, so again if you want to integrate with other Python software (i.e. Jupyter notebooks) you must install the other software inside the virtual environment of the engine. You can get the location of the engine virtual enviroment with the command
-```
-$ oq info venv
-```
 
-Scientists wanting to develop new GMPEs or new features for the engine should **[Install the OpenQuake Engine for development](universal.md)**.
+### Contents
 
-### PyPI
+* [Hardware requirements](#hardware-requirements)
+* [Installing the Long Term Support (LTS) version](#installing-the-long-term-support-lts-version)
+* [Installing the latest version](#installing-the-latest-version)
+* [Changing the OpenQuake Engine version](#changing-the-openquake-engine-version)
+* [Other installation methods](#other-installtion-methods)
+* [Getting help](#getting-help)
 
-The OpenQuake Engine is also available on **[PyPI](https://pypi.python.org/pypi/openquake.engine)** and can be installed in any Python 3 environment via `pip`:
 
-```
-$ pip install -r https://raw.githubusercontent.com/gem/oq-engine/master/requirements-py38-linux64.txt openquake.engine
-$ pip install openquake.engine
-```
-This works for Linux and Python 3.8. You can trivially adapt the command to Python 3.7 and 3.6 and to other
-operating systems. For instance for Windows it would be
+## Hardware requirements
 
-```
-$ pip install -r https://raw.githubusercontent.com/gem/oq-engine/master/requirements-py38-win64.txt openquake.engine
-```
-and for Mac
-```
-$ pip install -r https://raw.githubusercontent.com/gem/oq-engine/master/requirements-py38-macos.txt openquake.engine
-```
+**Single computer**
+- 8 GB of RAM
+- + 1.2 GB of free disk space
+- Operating system:
+  - Windows 10 (64bit)
+  - macOS 10.15 (Catalina) or above
```

Comment:
Catalina is no longer supported by Apple, we should probably say macOS 11 (Big Sur) or later?

Reply:
done